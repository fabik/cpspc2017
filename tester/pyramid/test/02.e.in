1 50
2858
130016509588964
145319830786145
319515657587598
85536525011285
55488442008416
300621772947374
84820344130957
101648981898604
218182256988677
527868726664
42804686016188
43284011603858
213228424935619
139847192342803
253607216497461
109614020355467
46858194531521
160606133254195
313272054783901
259901448757181
204660518214652
192469610816014
149228540631647
269153586543392
41521425064845
249536374414865
269736271495294
212201902396880
187141094126138
219048248803643
55671477662751
139424775372634
103381221456190
320672149116328
15735393475245
174856117690685
199966610277883
169137708319257
237860480634862
30008184738207
265005807665138
208058441676274
234945237518527
158683174970751
153076927143758
155927138128010
156507851423275
5583725023827
198858515840775
288728871502636
