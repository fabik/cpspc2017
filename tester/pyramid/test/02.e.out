371587184405259112
415324076386802410
913175749385355084
244463388482252530
158585967260052928
859177027083594892
242416543526275106
290512790266210232
623564890473638866
1508648820805712
122335792634265304
123705705163826164
609406838465999102
399683275715730974
724809424749743538
313276870175924686
133920719971087018
459012328840489310
895331532572389058
742798340548023298
584919761057475416
550078147712168012
426495169125247126
769240950341014336
118668232835327010
713174958077684170
770906263933550252
606473037050283040
534849247012502404
626039895080811694
159109083160142358
398476008014987972
295463530921791020
916481002174465424
44971754552250210
499738784359977730
571504572174189614
483395570376436506
679805253654435596
85763391981795606
757386598306964404
594631026310791092
671473488827950166
453516514066406358
437493857776860364
445639760769852580
447299439367719950
15958286118097566
568337638272934950
825187114754533688
